api/src/certification/session-management/domain/usecases/process-auto-jury.js

```diff
@@ -72,12 +81,13 @@ async function _handleAutoJuryV2({
         certificationCourseId: certificationCourse.getId(),
       });
 
-      certificationJuryDoneEvents.push(certificationJuryDoneEvent);
+      await certificationRescoringRepository.execute({
+        event: certificationJuryDoneEvent,
+      });
     }
   }
 
   return {
```

Comment:
En rapport avec mon commentaire cote controller

```suggestion
  return new AutoJuryDone(...)
```